DOCS-3872 CI Results Explorer Documentation Edits (#14821)
DataDog/documentation: content/en/synthetics/explorer/search_syntax.md

---
title: Search Syntax
kind: documentation
description: Learn how to create a search query in the CI Results Explorer.
further_reading:
- link: '/synthetics/explorer'
  tag: 'Documentation'
  text: 'Learn about the CI Results Explorer'
---

> ## Overview

Reviewer: This is a really great suggestion to help customers with the query syntax!!

A query is composed of terms and operators. 

There are two types of terms:

- A **single term** is a single word such as `test` or `hello`.

> - A **sequence** is a group of wods surrounded by double quotes, such as `"hello dolly"`.

Reviewer 2: ```suggestion
- A **sequence** is a group of words surrounded by double quotes, such as `"hello dolly"`.
```

To combine multiple terms into a complex query, you can use any of the following Boolean operators:

| Operator | Description                                                                                        |
|--------------|------------------------------------------------------------------------------------------------------- |

> | `AND`        | **Intersection**: Both terms are in the selected views (if nothing is added, AND is taken by default). |

Reviewer 2: ```suggestion
| `AND`        | **Intersection**: Both terms are in the selected views. If no operator is used, `AND` is used by default. |
```
